src/components/Badge/Badge.styles.js

```diff
@@ -0,0 +1,40 @@
+import styled from "styled-components";
+
+export const TooltipWrapper = styled.div`
+    position: relative;
+    display: block;
+`;
+
+export const TooltipText = styled.span`
+    visibility: hidden;
```

Comment:
hey @roopeshsn,
I can see that you're using `visibility` to manage the visibility of the tooltip, which is good and gets it working. But the issue would be that the tooltip text would always remain in the UI and DOM and take the required place, it's just that it is not visible.
Can we do it using the `display` property by setting it to `none` and `block`?
What are your thoughts?

Reply:
Sure, I'll change it.